synapse/http/servlet.py

```diff
@@ -15,6 +15,7 @@
 """ This module contains base REST classes for constructing REST servlets. """
 
 import logging
+from typing import Iterable, List, Literal, Optional, Union, overload
```

Comment:
Ah, the test failures look to be due to old pythons not having `Literal` in `typing`.
```suggestion
from typing import Iterable, List, Optional, Union, overload
from typing_extensions import Literal
```